support/support_tokens/README.md

```diff
@@ -1,9 +1,13 @@
-This folder contains the public keys for our support PIV tokens.
+This folder contains the public keys for our support PIV tokens and online
+tech port unlock.
 
-`authorized_keys` is from [`oxidecomputer/support-piv-tokens@36abeb82`](https://github.com/oxidecomputer/support-piv-tokens/blob/36abeb82aacb3640be2f6c988e49844356cf65a8/authorized_keys).
+Most of `authorized_keys` is from [`oxidecomputer/support-piv-tokens@36abeb82`](https://github.com/oxidecomputer/support-piv-tokens/blob/36abeb82aacb3640be2f6c988e49844356cf65a8/authorized_keys),
+edited to remove spaces from the comment field; see [`rustcrypto/SSH#289`](https://github.com/RustCrypto/SSH/pull/289).
```

Comment:
Should we un-edit this, now that we've bumped `SSH` to support comments with spaces?

Reply:
The file in [omicron](https://github.com/oxidecomputer/omicron/blob/main/smf/switch_zone_setup/support_authorized_keys) already has spaces, so it looks like they now match!